Example/BugShaker/AppDelegate.swift

```diff
@@ -21,9 +21,17 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     *  and an optional custom subject line to use for all bug reports.
     */
 
-    BugShaker.configure(to: ["example@email.com"], subject: "Bug Report", body: "Hi Developers, I am reporting a bug where ... ")
+    BugShaker.configure(to: ["example@email.com"], subject: "Bug Report", body: "Hi Developers, I am reporting a bug where ... ", delegate: self)
     
     return true
   }
 
+    func shouldPresentReportPrompt() -> Bool {
+        // here you can read some settings from user defaults and decide if you want to have the bug report alert to show up or not
+        if NSUserDefaults.standardUserDefaults().stringForKey("CONFIG_ENABLE_BUGSHAKE") != "\(true)" {
```

Comment:
I generally tend to write everything as string to `NSUserDefaults`. Have not tested with Swift, but at least in Obj C, `bookForKey` returns `false` both when you have written `false` to the userdefaults and when you have nothing there (which should return nil in my opinion). and there are some other oddities. 
But if you have strong feelings about this, please feel free to either check it as string or use `bookForKey`.